glava/render.c

```diff
@@ -125,8 +125,8 @@ struct gl_data {
     struct gl_wcb* wcb;
     int lww, lwh, lwx, lwy; /* last window dimensions */
     int rate; /* framerate */
-    double tcounter;
-    int fcounter, fcounter_last, ucounter, kcounter;
+    float tcounter, tcounter_last;
```

Comment:
I made `tcounter` float, because double values are slower to compute with and require a GL extension on the shader side if I'm correct, so I decided on this.